static/app/constants/index.tsx

```diff
@@ -229,66 +228,58 @@ export const DEFAULT_RELATIVE_PERIODS_PAGE_FILTER = {
   '30d': t('30D'),
 };
 
-export const DATA_CATEGORY_NAMES = {
```

Comment:
@leeandher It looks like this constant is used in `getsentry`. You'd need to do a 2 or 3 step change to migrate and apply this change.

Reply:
The new version of it down below is actually identical, just using the `DATA_CATEGORY_INFO` const, but I think it's a good call to keep it identical -- since as Nathan pointed out, it's not used in sentry right now. I'll come back and delete it after changing the references in getsentry, but for now i'll leave it as is.